aii-core/src/main/eg/aii-shellfe.conf

```diff
@@ -15,3 +15,22 @@
 # The default is profile_ for backward compatibility
 # Comment this out when not prefix is needed
 profile_prefix = profile_
+
+# Directory on TFTP server where PXELINUX configuration files are.
+# Must be an absolute path.
+# nbpdir = /osinstall/nbp/pxelinux.cfg
+
+# Directory on TFTP server where Grub PXE configuration files are.
+# Must be an absolute path.
+# Default value is based on 'pxelinux' option if defined with last
+# path token replaced by 'grub-efi' or '/osinstall/nbp/grub-efi' otherwhise.
+# nbpdir_grub =
+
+
+# Parent directory for the installation kernel/initrd to use with Grub EFI.
+# Must be an absolute path relative to TFTP server root.
```

Comment:
"absolute path relative to"...? i know what you mean, but why not a relative path (and you can precede it with a `/`)